riff-raff.yaml

```diff
@@ -29,6 +29,11 @@ deployments:
       templateStagePaths:
         CODE: NewslettersTool-CODE.template.json
         PROD: NewslettersTool-PROD.template.json
+      templateParameters:
```

Comment:
Is it worth adding an inline comment here to explain what's happening in case folk come across it in its temporal state?

Reply:
Good idea, added a comment